### PR TITLE
refactor: complete metadata for sources, remove unused yml column metadata 

### PR DESCRIPTION
**Pull request by lawrenceadams**
- `conditions` >> DESCRIPTION column not declared
-  multiple columns declared in synthea staging models that are not used in dbt-synthea

Loosely related to #101 #98

### CODE REVIEW

File: models/staging/synthea/_synthea__models.yml [hunk 1]
```diff
@@ -26,8 +26,6 @@ models:
       - name: patient_suffix
         data_type: text
         description: "Name suffix, such as PhD, MD, JD, etc."
-      - name: patient_maiden_name
```
Comment (katy-sadowski) [inline models/staging/synthea/_synthea__models.yml:29]: ah i think this should actually be `maiden_name`

File: models/staging/synthea/_synthea__models.yml [hunk 2]
```diff
@@ -748,12 +746,6 @@ models:
       - name: uncovered_encounters
         data_type: integer
         description: "The number of Encounters not paid for by this Payer, and paid out of pocket by patients."
-      - name: covered_medications
-        data_type: integer
-        description: "The number of Medications paid for by this Payer."
-      - name: uncovered_medications
-        data_type: integer
-        description: "The number of Medications not paid for by this Payer, and paid out of pocket by patients."
```
Comment (katy-sadowski) [inline models/staging/synthea/_synthea__models.yml:756]: actually it looks like these should be in the stg table, i just forgot to include them in the sql query 😮‍💨

File: models/staging/synthea/_synthea__models.yml [hunk 3]
```diff
@@ -769,13 +761,7 @@ models:
       - name: unique_customers
         data_type: integer
         description: "The number of unique patients enrolled with this Payer during the entire simulation."
-      - name: avg_qols
-        data_type: double precision
-        description: "The average Quality of Life Scores (QOLS) for all patients enrolled with this Payer during the entire simulation."
-      - name: member_months
-        data_type: integer
-        description: "The total number of months that patients were enrolled with this Payer during the simulation and paid monthly premiums (if any)."
-
```
Comment (katy-sadowski) [inline models/staging/synthea/_synthea__models.yml:778]: more naming issues from me - in the query these exist but are `qols_avg` and `payer_member_months` (not really sure why i decided to add payer_ there haha, could probably remove)

**Review by katy-sadowski — CHANGES_REQUESTED** [review]
thanks for this!  i checked and the "extra" columns are actually ones i missed adding to the stg queries or messed up the naming between the 2 files.

File: models/staging/synthea/_synthea__models.yml (hunk 1 above)
Comment (lawrenceadams) [reply models/staging/synthea/_synthea__models.yml:29]: good catch!

File: models/staging/synthea/_synthea__models.yml (hunk 2 above)
Comment (lawrenceadams) [reply models/staging/synthea/_synthea__models.yml:756]: Potentially! I don't think they're used downstream (yet) though?

File: models/staging/synthea/_synthea__models.yml (hunk 3 above)
Comment (lawrenceadams) [reply models/staging/synthea/_synthea__models.yml]: Ahhh I see what you mean

File: models/staging/synthea/_synthea__models.yml (hunk 2 above)
Comment (katy-sadowski) [reply models/staging/synthea/_synthea__models.yml:756]: they're not, but for consistency i think we should just add them to the models (i also _believe_ this is the general recommendation from dbt)

Comment (lawrenceadams) [reply models/staging/synthea/_synthea__models.yml:756]: Ahh sorry I'm being thick - I see what you mean forgive me!

**Review by katy-sadowski — APPROVED** [review]
awesome thank you!

**Pull request merged**